CHANGELOG.md

```diff
@@ -3,6 +3,38 @@
 - [v1.0.0 - v1.9.10](CHANGELOG-pre-v1.10.md)
 - [v0.11.6 and earlier](CHANGELOG-v0.md)
 
+## 1.18.4
+### January 30, 2025
+
+CHANGES:
+
+* auth/cf: Update plugin to v0.19.1 [[GH-29295](https://github.com/hashicorp/vault/pull/29295)]
+* sdk: Updated golang and dependency versions to be consistent across core, API, SDK [[GH-29422](https://github.com/hashicorp/vault/pull/29422)]
+
+FEATURES:
+
+* **Skip auto import rotation of static roles (enterprise)**: The Database secrets engine now allows skipping the automatic rotation of static roles during import.
```

Comment:
If possible, we might want to move this to be an "improvement" as we don't tend to release features in minor releases.